lib/facter/git_username.rb

```diff
@@ -0,0 +1,4 @@
+Facter.add("git_username") do
+  confine :kernel => 'Linux'
+  setcode "/bin/awk '/name/{print $NF}' /home/centos/.gitconfig"
```

Comment:
@josephoaks I tried this command on my gitconfig with the line `name = Greg Larkin` and the result was `Larkin`. Should the command handle first name and last name?